build/linux/installer/scripts/tomlparser-health-config.rb

```diff
@@ -13,32 +13,83 @@
 @configMapMountPath = "/etc/config/settings/agent-settings"
```

Comment:
please rename this file to agent-settings

Reply:
Done.